modules/ting/ting.module

```diff
@@ -439,23 +439,23 @@ function ting_entity_info() {
  * Implements hook_ding_entity_is().
  */
 function ting_ding_entity_is($object, $class) {
-  if ($class == 'reservable' || $class == 'cartable') {
-    $sources = variable_get('ting_reservable_sources', _ting_default_reservable_sources());
-    $types = variable_get('ting_reservable_types', _ting_default_reservable_types());
-    if (in_array(drupal_strtolower($object->ac_source), $sources) && in_array(drupal_strtolower($object->type), $types)) {
-      return TRUE;
-    }
-    else {
-      return FALSE;
-    }
-  }
-  elseif ($class == 'periodical') {
-    // TODO BBS-SAL: use a (new) provider call - hardcode the return-value in
-    // the opensearch implementation.
-    // @todo make this configurable.
-    // @todo also make sure that availability information isn't displayed
-    //   on the object
-    return in_array(drupal_strtolower($object->type), array('tidsskrift', 'periodikum', 'årbog'));
+  switch ($class) {
+    case 'reservable':
+    case 'cartable':
+      $sources = variable_get('ting_reservable_sources', _ting_default_reservable_sources());
+      $types = variable_get('ting_reservable_types', _ting_default_reservable_types());
+      return (in_array(drupal_strtolower($object->ac_source), $sources) && in_array(drupal_strtolower($object->type), $types));
+
+    case 'periodical':
+      // TODO BBS-SAL: use a (new) provider call - hardcode the return-value in
+      // the opensearch implementation.
+      // @todo make this configurable.
+      // @todo also make sure that availability information isn't displayed
+      //   on the object
+      return in_array(drupal_strtolower($object->type), array('tidsskrift', 'periodikum', 'årbog'));
+
+    case 'online':
+      return !empty($object->online_url);
```

Comment:
The merge in effect reverted this back to !empty($object->online_url);

Think it should be  
$object->getOnline_url()